modules/oc-mirror-procedure-delete-v2.adoc

```diff
@@ -61,8 +58,14 @@ $ oc mirror delete --config delete-image-set-config.yaml --workspace file://<pre
 where:
 
 <previously_mirrored_work_folder>:: Specifies the directory where images were previously mirrored to or stored during the mirroring process.
+
 <remote_registry>:: Specifies the URL or address of the remote container registry from which images will be deleted.
 
```

Comment:
Also, add a + here so that this admonition doesn't break the number order of the steps:
```suggestion
+
```